code/modules/hydroponics/grown/nettle.dm

```diff
@@ -48,9 +48,11 @@
 	throw_range = 3
 	attack_verb_continuous = list("stings")
 	attack_verb_simple = list("sting")
+	var/minforce = 4 //minimum force after most of the stingy bits fall off. increases with potency.
+	var/bareicon = "nettle_bare" //icon used to represent nettle once all its bonus damage is used up
```

Comment:
this is not autodoc'd correctly, use three slashes and put it above the variables
```
/// comment
var/var
```